### DISCUSSION TITLE
Create a Discord server for the org

### DISCUSSION THREAD

**Discussion by priyankarpal** [ExhibitHQ/ExhibitHQ-old#2] [General]
We can create a discord server for this org where we can discuss A-Z well. 

What do you think devs?

@Arindam200 @samyabrata-maji @Swpn0neel

**Comment by sammaji** [comment]
Good idea 💡

But is it good to start a discord community without actually having a finished project?

**Reply by priyankarpal** [reply]
create a Private discord for the community now. We will public it after completing the project. 

what do you think?

**Reply by Swpn0neel** [reply]
Sounds good to me, please share your Discord I'd @priyankarpal

**Reply by priyankarpal** [reply]
Discord id: `Priyankar Pal#9159`
